lib/private/Settings/Manager.php

```diff
@@ -111,7 +109,7 @@ public function registerSection(string $type, string $section) {
 	}
 
 	/**
-	 * @param string $type 'admin' or 'personal'
+	 * @psalm-param 'admin'|'personal' $type
```

Comment:
Does that work in psalm:
```suggestion
	 * @psalm-param self::KEY_*_SECTION $type
```

Reply:
No, seems like the asterisk is only allowed at the end